prestapyt/prestapyt.py

```diff
@@ -86,10 +90,11 @@ class PrestaShopWebService(object):
     """Interact with the PrestaShop WebService API, use XML for messages."""
 
     MIN_COMPATIBLE_VERSION = '1.4.0.17'
-    MAX_COMPATIBLE_VERSION = '1.7.5.2'
+    # 4th version number is to avoid constant version changes
+    MAX_COMPATIBLE_VERSION = '1.7.8.999'
 
     def __init__(self, api_url, api_key, debug=False, session=None,
-                 verbose=False):
+                 verbose=False, custome_headers={}):
```

Comment:
```suggestion
                 verbose=False, custom_headers=None):
```

Reply:
Ups!. Fixed

Reply:
Sorry, if I use default custom_headers=None and I do an instance without fill this param, it is throw an Exception. This would imply that if we upload this new version we would break the Prestashop connector with Odoo